app.rb

```diff
@@ -0,0 +1,99 @@
+require_relative 'book'
+require_relative 'person'
```

Comment:
* Kindly fix this error I get when I try to run your app `superclass must be an instance of Class (given an instance of Module) (TypeError)`  so the error is coming from the nameable which should be a class but instead a module hence the error, below is a screenshot of the said error 
![image](https://github.com/Aqurds/OOP-school-library/assets/52497246/a5ff5f5d-53c5-4af3-88b3-023710a49e0e)

Reply:
@Davidosky007 can you please suggest me a quick fix on this?